src/index.js

```diff
@@ -18,7 +27,7 @@ async function doScript() {
         ['config', 'minimum-stability', 'dev'],
         ['config', 'prefer-stable', 'true'],
         ['config', 'preferred-install', 'dist'],
-        ['config', 'repositories.0', "{\"type\": \"path\", \"url\": \"$GITHUB_WORKSPACE\", \"options\": {\"symlink\": false}}"],
+        ['config', 'repositories.0', `{"type": "path", "url": "${githubWorkspacePath}", "options": {"symlink": false}}`],
         ['config', 'repositories.1', 'composer', 'https://packages.drupal.org/8']
```

Comment:
For some reason this command is not executing. It skips this, then the next. And goes to the prophecy-phpunit